operator-framework/pom.xml

```diff
@@ -65,7 +59,6 @@
     <dependency>
       <groupId>io.fabric8</groupId>
       <artifactId>crd-generator-apt</artifactId>
-      <scope>test</scope>
```

Comment:
The CRD generator is just required for the tests, is it required to remove this? (I guess not)

Reply:
Removed that while experimenting. I'll put it back!